manifests/hourly.pp

```diff
@@ -12,30 +12,27 @@
 #   class { 'logrotate::hourly':
 #     ensure => absent,
 #   }
-class logrotate::hourly(
-  $ensure = 'present',
+class logrotate::hourly (
+  Enum['present','absent'] $ensure='present'
 ) {
 
-  case $ensure {
-    'absent': {
-      $dir_ensure = $ensure
-    }
-    'present': {
-      $dir_ensure = 'directory'
-    }
-    default: {
-      fail("Class[Logrotate::Hourly]: Invalid ensure value '${ensure}'")
-    }
+  $dir_ensure = $ensure ? {
+    'absent'  => $ensure,
+    'present' => 'directory'
   }
 
-  file { "${logrotate::rules_configdir}/hourly":
-      ensure => $dir_ensure,
-      owner  => $logrotate::root_user,
-      group  => $logrotate::root_group,
-      mode   => '0755',
+  file { '/etc/logrotate.d/hourly':
```

Comment:
I think this hard-coded path will break on some distros. Any reason you changed this from `"${logrotate::rules_configdir}/hourly"` ?